packages/hooks/src/useTradingPlatformStatus.ts

```diff
@@ -9,14 +10,21 @@ export type TradingPlatformStatus = {
 /** A custom hook that gets the list of statuses of ctrader dxtrade mt5 platform. */
 const useTradingPlatformStatus = () => {
     const [data, setData] = useState<TradingPlatformStatus[] | null>(null);
+    const isMounted = useRef(true);
```

Comment:
there's a hook to check if the component is mounted in `@deriv/shared`. can we use it?

Reply:
I tried using `useIsMounted`, but it caused infinite API calls. This might be due to how the WebSocket subscription is being handled.